src/index.ts

```diff
@@ -242,10 +243,45 @@ async function wranglerCommands() {
 				}
 			}
 
-			await exec(`${packageManager.exec} wrangler ${command}`, args, {
+			// Used for saving the wrangler output
+			let stdOut = "";
+			let stdErr = "";
+
+			// Construct the options for the exec command
+			const options = {
 				cwd: config["workingDirectory"],
 				silent: config["QUIET_MODE"],
-			});
+				listeners: {
+					stdout: (data: Buffer) => {
+						stdOut += data.toString();
+					},
+					stderr: (data: Buffer) => {
+						stdErr += data.toString();
+					},
+				},
+			};
+
+			// Execute the wrangler command
+			await exec(`${packageManager.exec} wrangler ${command}`, args, options);
+
+			// Set the outputs for the command
+			setOutput("command-output", stdOut);
+			setOutput("command-stderr", stdErr);
+
+			// Check if this command is a workers or pages deployment
+			if (
+				command.startsWith("deploy") ||
+				command.startsWith("publish") ||
+				command.startsWith("pages publish")
```

Comment:
We also need to check for `pages deploy`

Reply:
Added!